apps/recorder/app.js

```diff
@@ -305,7 +305,7 @@ function viewTrack(filename, info) {
     g.drawString(require("locale").distance(dist),g.getWidth() / 2, g.getHeight() - 20);
     g.setFont("6x8",2);
     g.setFontAlign(0,0,3);
-    g.drawString("Back",g.getWidth() - 10, g.getHeight() - 40);
+    g.drawString("Back",g.getWidth() - 10, g.getHeight() - 40); // TODO: this position depends on Bangle1 vs 2
```

Comment:
Is this TODO still required?

Reply:
Yes, but it is just something I ran into with the `recorder` app while messing around and is unrelated to the `exstats` and `run` changes. When plotting a map, altitude, or speed of an existing track, the back button is displayed in the bottom right of the screen where it would be on a banglejs1.

Reply:
Thanks! That's a good point. I just fixed this in an update, so it might be worth removing this line from the PR